woof-code/rootfs-packages/firewall_ng/usr/sbin/firewall_ng

```diff
@@ -1448,7 +1448,7 @@ else
 	exit
```

Comment:
The commit message, "Update firewall_ng", doesn't explain what this does and how it works

Reply:
I have updated the commit message to indicate: Update firewall_ng to allow correct Cancel and Apply button function. Is this sufficient or what would you advise?